src/remote_adapter/README.md

```diff
@@ -0,0 +1,52 @@
+## How to build
+- Install all [dependencies](#dependencies)
+- Clone repository
+- Create build directory and get into created directory
+- Run the following command : `cmake <path to sources>`
+- Run `make`
+
+## Dependencies:
+ - [CMake](https://cmake.org/download/) : Download and install any release version > 3.11
+ - **GNU Make & C++ compiler** : Install with command `sudo apt-get install build-essential`
+ - [rpclib](https://github.com/rpclib/rpclib) : Library for TCP communication.
+   *Note* : Installed automatically while running CMake.
+
+ - **Lua library** : Install with command `sudo apt-get install liblua5.2-dev`
+ - **Doxygen** : Install with command `sudo apt-get install doxygen`
+
+## Remote Testing Adapter server (RemoteTestingAdapterServer) usage
+ - Accepts rpclib connections
+ - Executes client requests using plugins
+
+libRemoteUtilsManager.so:
+`app_start` - start application on remote host
```

Comment:
This section doesn't show up as expected in https://github.com/smartdevicelink/sdl_atf/blob/feature/remote_atf_connection/src/remote_adapter/README.md#remote-testing-adapter-server-remotetestingadapterserver-usage. Each of these actions should be in a separate bullet point
```suggestion
- `app_start`: start application on remote host
```

Reply:
@ShobhitAd Corrected in 97ff9fd